tests/Aspire.Microsoft.EntityFrameworkCore.SqlServer.Tests/SqlServerContainerFixture.cs

```diff
@@ -0,0 +1,36 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using Aspire.Hosting;
+using Testcontainers.MsSql;
+using Xunit;
+
+namespace Aspire.SqlServer.Tests;
+
+public sealed class SqlServerContainerFixture : IAsyncLifetime
```

Comment:
(nit) The PostgreSQLContainerFixture goes in the non-EF tests and then the EF tests links to it. Can we follow that same pattern here? Have the EF thing depend on the non-EF thing seems like the right layering.

Reply:
Would that be `Aspire.Microsoft.Data.SqlClient.Tests`?